### PR TITLE
etc: remove unused script and update readme

### LINKED ISSUE #267: run_etcd.sh has drifted from run_etcd_docker.sh
If the cluster is started up with the apparently older `run_etcd.sh`, it will not run the `etcd-ssl` or `etcd-proxy` processes, which are depended on by the maven tests.  So the tests fail when not using `run_etcd_docker.sh`.

I think they should be updated to match, or if `run_etcd.sh` is obsolete it should be removed.

### PR DESCRIPTION
**Pull request by fanminshi**
fixes #267

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @lburgazzoli

File: README.md [hunk 1]
```diff
@@ -77,56 +77,44 @@ The current major version is zero (0.y.z). Anything may change at any time. The
 
 ## Running tests
 
-The project is to be tested against a three node `etcd` setup, launched by the [scripts/run_etcd.sh](etc/scripts/run_etcd.sh) shell script:
+The project is to be tested against a three node `etcd` setup, launched by the [scripts/run_etcd_docker.sh](etc/scripts/run_etcd_docer.sh) shell script:
```
Comment (lburgazzoli) [inline README.md]: typo: run_etcd_docker.sh ?

File: README.md [hunk 2]
```diff
@@ -77,56 +77,44 @@ The current major version is zero (0.y.z). Anything may change at any time. The
...
 
 ```
-./etc/scripts/run_etcd.sh
+./etc/scripts/run_etcd_etcd.sh
```
Comment (lburgazzoli) [inline README.md]: typo: ./etc/scripts/run_etcd_docker.sh

**Comment by fanminshi** [comment]
all fixed!

**Pull request merged**